spec/ models/nutrient_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Nutrient, type: :model do
+  describe "validations" do
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:health_benefits) }
```

Comment:
Interesting! I don't think I've seen this syntax before for these validations! Is this a new/older syntax from shoulda-matchers? Looks good though!